sources/packages/backend/libs/sims-db/src/entities/notification.model.ts

```diff
@@ -165,4 +165,36 @@ export enum NotificationMessageType {
    * Ministry disbursement blocked.
    */
   MinistryNotificationDisbursementBlocked = 17,
+  /**
+   * Student submitted change request after COE.
+   */
+  StudentSubmittedChangeRequestNotificationForMinistry = 18,
```

Comment:
I do not see a reason to have every notification appended with the `ForMinistry`. Maybe it can be expressed in the comment.